sdk/core/azure-core/CMakeLists.txt

```diff
@@ -117,7 +119,7 @@ create_code_coverage(core azure-core azure-core-test)
 target_link_libraries(azure-core INTERFACE Threads::Threads)
 
 if(MSVC)
-    target_link_libraries(azure-core PRIVATE crypt32)
+    target_link_libraries(azure-core PRIVATE bcrypt crypt32)
 else()
     message("Find the OpenSSL package on MacOS, used for Base64 encoding.")
```

Comment:
```suggestion
    message("Find the OpenSSL package on macOS, used for Base64 encoding.")
```

Reply:
Unrelated to this PR, so leaving it as is for now.

Reply:
No longer relevant since the comment was removed in https://github.com/Azure/azure-sdk-for-cpp/pull/1620#discussion_r572217594